core/app/models/mno_enterprise/impac/dashboard.rb

```diff
@@ -44,10 +44,7 @@ def filtered_widgets_templates
     end
 
     def to_audit_event
-      {
-        name: name,
-        organization_id: (owner_type == 'Organization') ? owner_id : nil
-      }
+      { name: name }
```

Comment:
@cesar-tonnoir didn't spot this one during code review. Why was this removed? I think that'd introduce a regression (see MNOE-861)

Reply:
@ouranos I think this is because after this change, owner_type should never be `Organization` again, so `organization_id` would always be nil - I figured out it would be more relevant to simply delete the field?...